handler.go

```diff
@@ -162,11 +162,45 @@ func withHTTPContext(ctx context.Context) context.Context {
 	return WithContext(ctx, hctx)
 }
 
+func setTracingContext(config map[string]string, header http.Header) tracingCtx {
+	if config["OCI_TRACING_ENABLED"] == "0" {
+		// When tracing is not enabled then we
+		// assign empty tracing context to
+		// the context
+		return tracingCtx{}
+	}
+	tctx := tracingCtx{
+		traceCollectorURL: config["OCI_TRACE_COLLECTOR_URL"],
+		traceId:           header.Get("x-b3-traceid"),
```

Comment:
Does this support canonicalised headers too? The Node FDK one does.

Reply:
Not sure but using the go http library